packages/web-react/src/components/UNSTABLE_Toggle/demo/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import DocsSection from '../../../../docs/DocsSections';
+import ToggleDefault from './ToggleDefault';
+import ToggleDisabled from './ToggleDisabled';
+import ToggleFluid from './ToggleFluid';
+import ToggleHelperText from './ToggleHelperText';
+import ToggleHiddenLabel from './ToggleHiddenLabel';
+import ToggleIndicators from './ToggleIndicators';
+import ToggleRequired from './ToggleRequired';
+import ToggleValidation from './ToggleValidation';
+
+ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+  <React.StrictMode>
+    <DocsSection title="Default" stackAlignment="stretch">
+      <ToggleDefault />
+    </DocsSection>
+    <DocsSection title="Indicators" stackAlignment="stretch">
+      <ToggleIndicators />
+    </DocsSection>
+    <DocsSection title="Required" stackAlignment="stretch">
+      <ToggleRequired />
+    </DocsSection>
+    <DocsSection title="Hidden Label" stackAlignment="stretch">
+      <ToggleHiddenLabel />
+    </DocsSection>
+    <DocsSection title="Fluid" stackAlignment="stretch">
+      <ToggleFluid />
+    </DocsSection>
+    <DocsSection title="Disabled" stackAlignment="stretch">
+      <ToggleDisabled />
+    </DocsSection>
+    <DocsSection title="HelperText" stackAlignment="stretch">
```

Comment:
```suggestion
    <DocsSection title="Helper Text" stackAlignment="stretch">
```
and please update visual test again